Fabric.Terminology.API/Scripts/Install-Terminology.ps1

```diff
@@ -1,90 +1,19 @@
 )
 
-if([string]::IsNullOrEmpty($moduleFilePath)){
-	Install-Module DosInstallUtilities -Scope CurrentUser
-
-	Import-Module DosInstallUtilities -Force
-}
-else{
-	Import-Module $moduleFilePath -Force
-}
-
-if(!(Test-Path .\Fabric-Install-Utilities.psm1)){
-    Invoke-WebRequest -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -Headers @{"Cache-Control"="no-cache"} -OutFile Fabric-Install-Utilities.psm1
-}
-Import-Module -Name .\Fabric-Install-Utilities.psm1 -Force
-
-$installSettings = Get-InstallationSettings "terminology"
-
-[string] $credentialUsername = $installSettings.iisUser
-
-#Todo: maybe we should prompt and not pick this up from the config file?
-[securestring] $credentialPassword = ConvertTo-SecureString $installSettings.iisUserPwd -AsPlainText -Force
-[PSCredential] $identityCredentials = New-Object System.Management.Automation.PSCredential ($credentialUsername, $credentialPassword)
-
-#Not sure if we want to fetch the zip file from the config - if so, remove as a param.
-Publish-DosWebApplication -WebAppPackagePath $applicationZipFile -AppPoolName $installSettings.appPool -AppPoolCredential $identityCredentials -AppName $installSettings.appName -IISWebSite $installSettings.siteName
-
-$configurationReplacementTargets = @("**SqlServerConnectionString**","**DiscoveryServiceUri**")
-
-$configurationReplacementValues = @($installSettings.sqlServerAddress, $installSettings.discoveryService)
-
-$terminologyWebInstallFolder = Get-IISWebSitePath -WebSiteName $installSettings.siteName
-
-Update-DosConfigFile -TargetPatterns $configurationReplacementTargets -ReplacementPattern $configurationReplacementValues -Delimiter "**" -FilePath "$terminologyWebInstallFolder\$($installSettings.appName)\appsettings.json"
-
-
-<# TODO: Ben create Terminology shared db role for iis user
-
-Shared Database Role
-        
-VIEW    Terminology.Code                    Read
-VIEW    Terminology.CodeSystem              Read
-VIEW    Terminology.ValueSetCode            Read
-VIEW    Terminology.ValueSetCodeCount       Read
-VIEW    Terminology.ValueSetDescription     Read
-TABLE   ClientTerm.ValueSetCodeBASE         Read/Write
-TABLE   ClientTerm.ValueSetCodeCountBASE    Read/Write
-TABLE   ClientTerm.ValueSetDescriptionBASE  Read/Write
+Import-Module "$PSScriptRoot\Terminology-Intall-Utilities.psm1" -Force
 
-#>
+Import-Modules $moduleFilePath
 
-<# TODO: Ben register Terminology with auth/identity
+$config = Get-TerminologyConfig $applicationZipFile
 
-Fabric.Identity
+Publish-DosWebApplication -WebAppPackagePath $applicationZipFile -AppPoolName $config.appPool -AppPoolCredential $config.iisUserCredentials -AppName $config.appName -IISWebSite $config.siteName
 
-	ApiResource:	terminology-api
-	Scopes:			fabric/terminology.read, fabric/terminology.write
-	Client:			terminology-client
-	Scopes:			fabric/authorization.read
-	Grant-types:	delegation, client_credentials
+Update-AppSettings $config
```

Comment:
This seems like a potential candidate to promote to DosInstallUtilities - seems like a lot of .net core web apps may want to use something similar to update the settings file on install.